src/ast/nodes/Property.ts

```diff
@@ -81,16 +82,16 @@ export default class Property extends NodeBase {
 					options.getHasEffectsWhenCalledOptions()
 				) ||
 				(!options.hasReturnExpressionBeenAccessedAtPath(path, this) &&
-					this.value.someReturnExpressionWhenCalledAtPath(
-						[],
-						this._accessorCallOptions,
-						innerOptions => node =>
-							node.hasEffectsWhenAccessedAtPath(
-								path,
-								innerOptions.addAccessedReturnExpressionAtPath(path, this)
-							),
-						options
-					))
+				 this.value.someReturnExpressionWhenCalledAtPath(
+					 [],
+					 this._accessorCallOptions,
+					 innerOptions => node =>
+						 node.hasEffectsWhenAccessedAtPath(
+							 path,
+							 innerOptions.addAccessedReturnExpressionAtPath(path, this)
+						 ),
+					 options
+				 ))
```

Comment:
Were these tab -> space conversions intended?

Reply:
I guess not :) I think I need to do the "prettier" PR soon.